packages/ui/src/components/select.tsx

```diff
@@ -5,23 +5,41 @@ import { Check, ChevronDown } from "lucide-react";
 import * as React from "react";
 
 import { cn } from "../lib/utils";
+import { cva, VariantProps } from "class-variance-authority";
 
 const Select = SelectPrimitive.Root;
 
 const SelectGroup = SelectPrimitive.Group;
 
 const SelectValue = SelectPrimitive.Value;
 
+const selectVariants = cva(
+  "flex w-full items-center justify-between px-3 text-sm focus:outline-none focus:ring-2 focus:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50",
+  {
+    variants: {
+      variant: {
+        default:
+          "border-input h-10 ring-offset-background py-2 placeholder:text-muted-foreground focus:ring-ring rounded-md border bg-transparent ",
+
+        ghost: "focus:ring-accent focus:ring-offset-accent space-x-2",
```

Comment:
It's a bit messy. E.g. only having a height in 'default' but not in 'ghost' variant.

Personally, I'd just move the classes `border-none text-muted-foreground' to the props of the SelectTrigger and that's it.

Reply:
but i don't want to have so much spacing next the arrow :(